rules/no-duplicate-literal-elements.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+const {newExpressionSelector} = require('./selectors/index.js');
+
+const MESSAGE_ID = 'no-duplicate-literal-elements';
+const messages = {
+	[MESSAGE_ID]: 'Remove duplicate {{valueType}} `{{value}}` from the `{{objectType}}`.',
+};
+
+const arraySelector = [
+	'VariableDeclarator > ArrayExpression > Literal',
+	'ExpressionStatement > ArrayExpression > Literal',
+	'AssignmentExpression > ArrayExpression > Literal',
+].join(',');
+
+const setSelector = [
+	newExpressionSelector('Set'),
+	'[arguments.0.type="ArrayExpression"]',
+	' > ',
+	'ArrayExpression > Literal',
+].join('');
+
+const mapSelector = [
+	newExpressionSelector('Map'),
+	'[arguments.0.type="ArrayExpression"]',
+	' > ',
+	'ArrayExpression > ArrayExpression > Literal:first-child',
+].join('');
+
+const checkArrayHasDuplicatedValue = array => array.filter((element, index, array_) => array_.indexOf(element) !== index);
```

Comment:
This function can directly use `array.find` to find the duplicated element, so we don't need to do `array.includes()`.

Reply:
> This function can directly use `array.find` to find the duplicated element, so we don't need call `array.includes()` and `getFirstDuplicateIndex()`.

Oh! I see now.
Ok, I will try it later.

Reply:
I have refactored.
thx for your suggestion.